source/whats-new.txt

```diff
@@ -4,4 +4,139 @@
 What's New
 ==========
 
-.. TODO
\ No newline at end of file
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 5.1.3 <kotlin-sync-version-5.1.3>`
+* :ref:`Version 5.1.2 <kotlin-sync-version-5.1.2>`
+* :ref:`Version 5.1.1 <kotlin-sync-version-5.1.1>`
+* :ref:`Version 5.1 <kotlin-sync-version-5.1>`
+* :ref:`Version 5.0 <kotlin-sync-version-5.0>`
+
+.. _kotlin-sync-version-5.1.3:
+
+What's New in 5.1.3
+-------------------
+
+The 5.1.3 driver patch release includes the following changes:
+
+.. sharedinclude:: dbx/jvm/v5.1.3-wn-items.rst
+
+.. _kotlin-sync-version-5.1.2:
+
+What's New in 5.1.2
+-------------------
+
+The 5.1.2 driver patch release includes the following changes:
+
+- Support for encoding Kotlin data classes with nullable
+  generic parameter types. For example, you can encode the ``Container`` class
+  in the following code:
+  
+  .. code-block:: kotlin
+
+     @Serializable
+     data class Box<T>(
+         val boxed: T
+     )
+     
+     @Serializable
+     data class Container(
+         val box: Box<String?>
+     )
+
+.. _kotlin-sync-version-5.1.1:
+
+What's New in 5.1.1
+-------------------
+
+The 5.1.1 driver patch release includes the following changes:
+
+- When using the ``MONGODB-OIDC`` authentication mechanism, you must not
+  include comma characters in the ``authMechanismProperties`` connection
+  string value. To learn more about this behavior, see
+  `MongoCredential <{+core-api+}/com/mongodb/MongoCredential.html>`__ in the 
+  API documentation.
```

Comment:
S: on second thought, i think you should remove the last sentence here. The API documentation doesn't really describe this behavior, so it might be unhelpful to link there. It'll just be something we can add back in when we create the OIDC documentaion. If you want, you can add a TODO comment that says something like "Link to OIDC content" or something!